### PR TITLE
Api

### PR DESCRIPTION
**Pull request by em3rkof**

### CODE REVIEW

**Comment by ericingram** [comment]
Assuming this is still WIP, can you put WIP prefix on the title?

File: index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const Customers = require('./lib/api/customers');
+const Invoices = require('./lib/api/invoices');
+const Events = require('./lib/api/events');
+const Orders = require('./lib/api/orders');
+const Payments = require('./lib/api/payments');
+const Payouts = require('./lib/api/payouts');
+const Refunds = require('./lib/api/refunds');
+const Reimbursements = require('./lib/api/reimbursements');
+
+class ResolveNode {
```
Comment (ericingram) [inline index.js]: Can we change this to just `Resolve`? I think the `Node` part is obvious :)

File: lib/services/api.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+const fetch = require('cross-fetch');
+const qs = require('qs');
+
+const { toBase64 } = require('../util/crypto');
+const { errorTypes } = require('../util/error');
+
+class Api {
```
Comment (ericingram) [inline lib/services/api.js]: We should use all caps since it's an abbreviation -- `API`

File: lib/services/api.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  password: null,
+};
+
+Api._headers = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json',
+  'Pragma': 'no-cache',
+};
+
+Api._apiUrl = 'http://localhost:3000/api/v2';
```
Comment (ericingram) [inline lib/services/api.js]: This should default to production URL I think

File: lib/services/api.js (hunk 2 above)
Comment (ericingram) [inline lib/services/api.js]: Let's add a method to `setVersion` which would pass a header to set the API version according to the versioning scheme we implemented

Comment (em3rkof) [reply lib/services/api.js]: @ericingram done

File: lib/services/api.js (hunk 3 above)
Comment (em3rkof) [reply lib/services/api.js]: @ericingram done

File: lib/services/api.js (hunk 2 above)
Comment (em3rkof) [reply lib/services/api.js]: @ericingram done

File: index.js (hunk 1 above)
Comment (em3rkof) [reply index.js]: @ericingram done

**Comment by em3rkof** [comment]
@ericingram added MIT license file

**Pull request merged**